testhelpers/org.eclipse.n4js.tests.helper/src/org/eclipse/n4js/xpect/validation/suppression/SuppressIssuesSetup.xtend

```diff
@@ -45,9 +45,7 @@ import org.eclipse.xpect.xtext.lib.tests.ValidationTestModuleSetup.TestingResour
 @XpectImport( #[SuppressIssuesSetupRoot])
 class SuppressIssuesSetup extends IssuesByLineProvider {
 
-	private final Collection<String> suppressedIssueCodes = new ArrayList<String>(
-		N4JSLanguageConstants.DEFAULT_SUPPRESSED_ISSUE_CODES_FOR_TESTS
-	);
+	private final Collection<String> suppressedIssueCodes = new ArrayList<String>(this.defaultSuppressedIssueCodes);
```

Comment:
in Xtend you can just use `#[ ... ]`

Reply:
I know, but this translates to an immutable collection and below `suppressedIssueCodes` must be mutable in order to allow for the configuration on a per-file basis.